cmd/up/exec_unix.go

```diff
@@ -0,0 +1,41 @@
+//go:build !windows
+// +build !windows
```

Comment:
This file is going to be ignored when built for Windows using this directive, however below at line 17, we check for Windows - perhaps this line is a left over of previous attempts?